go-chaos/cmd/worker_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 Camunda Services GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"io"
+	"os"
+	"testing"
+)
+
+func SetStdOut(out *os.File) {
+	os.Stdout = out
+}
+
+func Test_ShouldRunCommand(t *testing.T) {
```

Comment:
:upside_down_face: I think this test is not adding much value. `runZbChaosCommand` doesn't do much other than just delegating to the cobra command.

Reply:
I would like to keep it. With this unit test failing, I can faster pin-point where an issue is, instead of having only a failed integration test.